Deploy-CAM.ps1

```diff
@@ -1499,11 +1497,12 @@ function New-ConnectionServiceDeployment() {
                 -Verbose
         }
         else {
-            for($idx = 10;$idx -gt 0;$idx--)
+            $maxRetries = 30
+            for($idx = 0;$idx -lt $maxRetries;$idx++)
             {
                 try {
                     New-AzureRmResourceGroupDeployment `
-                        -DeploymentName "CS" `
+                        -DeploymentName "CS$connectionServiceNumber-$idx" `
```

Comment:
If the deployment name is not unique then the deployment that was going to succeed will fail.